### DISCUSSION TITLE
Please double check Session ID for 2023 Heineken Dutch Grand Prix - Actual Race?

### DISCUSSION THREAD

**Discussion by adamhott** [br-g/openf1#81] [Q&A] [answered]
Hi, Wondering if you could double-check the Session ID for the 2023 Heineken Dutch Grand Prix - Actual Race?

I'm not seeing a formation lap, also in the race tracker on the formula1.com website, it doesn't match my simulation at all.

I'm wondering if I'm using the wrong data set. I'm using session id = 9149.

Thanks!

**Comment by adityakotha03** [comment, marked as answer]
Hey, I believe that the OpenF1 API in the year 2023 did not include a formation lap for at least some of the data like the stints; this was included in the 2024 data. I don't know why this was the case. I believe 9149 this is the correct ID for that race.
